docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.mdx

```diff
@@ -53,35 +58,90 @@ The icon should be somewhat in line with the contemporary icon styles (i.e. not
 
 The name should ideally be no longer than 15 characters, and must be shorter than 20 characters.
 
+:::danger Bad examples
+
+- Test app for testing stuff
```

Comment:
I'd go with some more realistic examples here, e.g. "Edit markdown in a calm, distraction-free environment and export to many differents formats"

Reply:
Ah wait nvm, I thought this was for summaries. Still, morerealistic examples would be better, e.g. "Apostrophe Markdown" and "Minimal Markdown Editor"?